src/lib/wdi5-fe.ts

```diff
@@ -24,10 +25,19 @@ function createProxy(myObj: any, type: string, methodCalls: any[], pageKeys: str
 }
 export class WDI5FE {
     constructor(private appConfig: any, private browserInstance: any) {}
-    static async initialize(appConfig, browserInstance = browser) {
-        await loadFELibraries(browserInstance)
-        await initOPA(appConfig, browserInstance)
-        return new WDI5FE(appConfig, browserInstance)
+    static async initialize(appConfig) {
```

Comment:
similar here - why remove `browserInstance` as parameter and instead break scope of the otherwise "pure" function?

Reply:
we never passed the second parameter to the `initialize` function. We only passed the config and just took default for `browserInstance`

We could stick to that pattern but then we have to pass the relevant browserInstance for the multiRemote in the `before` hook:

```javascript
before(async () => {
        FioriElementsFacade = await browser.fe.initialize({appconfig}, browserInstance)
})
```

I guess we just have to decide what is cleaner